### PR TITLE
added-scroll-to-top

### PR DESCRIPTION
**Pull request by rushabhpanbude**
added scroll to top feature

## Description
added scroll to top feature

## Screenshots
C4pture

### CODE REVIEW

File: src/CSS/ScrollToTop.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    border: none;
+    cursor: pointer;
+    display: none; /* Initially hide the button */
+    z-index: 9999;
+  }
+  
+.visible {
+    display: block; /* Show the button when the visible class is applied */
+}
+
```
Comment (priyankarpal) [inline src/CSS/ScrollToTop.css:18]: we are using Tailwind CSS so please use Tailwind CSS

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/CSS/ScrollToTop.css (hunk 1 above)
Comment (rushabhpanbude) [reply src/CSS/ScrollToTop.css:18]: removed css and used Tailwind CSS

**Review by priyankarpal — CHANGES_REQUESTED** [review]
add a default bg color 

image

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**